dev/com.ibm.ws.microprofile.openapi.ui/resources/com/ibm/ws/microprofile/openapi/ui/resources/OpenAPIUIMessages.nlsprops

```diff
@@ -0,0 +1,34 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWWKO
+#COMPONENTNAMEFOR Open API UI
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_NONE
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+
+# Message prefix block: CWWKO1750 - CWWKO1799
+
+CWWKO1750W_OPEN_API_UI_PATH_RESERVED=CWWKO1750W: The OpenAPI UI Path cannot be set to {0}. This Path is reserved for use by the application server.
+CWWKO1750W_OPEN_API_UI_PATH_RESERVED.explanation=The OpenAPI UI Path cannot be set to a value that has been reserved for use by the application server.
+CWWKO1750W_OPEN_API_UI_PATH_RESERVED.useraction=Specify a valid value for the extensions attribute in the Micro Profile configuration.
+
+CWWKO1751W_OPEN_API_UI_PATH_INVALID=CWWKO1751W: The OpenAPI UI Path cannot be set to {0}. The Path is invalid or contains invalid characters.
+CWWKO1751W_OPEN_API_UI_PATH_INVALID.explanation=The OpenAPI UI Path cannot be set to a value containing invalid characters.
+CWWKO1751W_OPEN_API_UI_PATH_INVALID.useraction=Specify a valid value for the extensions attribute in the configuration.
+
+CWWKO1752W_OPEN_API_UI_PATH_POTENTIAL_CONFLICT=CWWKO1752W: The OpenAPI UI Path has been set to the same path as the {0} feature and may prevent startup of itself or the other feature.
```

Comment:
```suggestion
CWWKO1752W_OPEN_API_UI_PATH_POTENTIAL_CONFLICT=CWWKO1752W: Setting the OpenAPI UI Path to the same path as the {0} feature might disrupt the startup process of itself or the other feature.
```

Reply:
Suggested change:
The OpenAPI UI Path was set to the same path as the {0} feature, which might prevent startup of itself or the other feature.

Reply:
suggested changes made